### ISSUE TITLE
Study normalization methods for deep learning

### ISSUE THREAD

**Issue by araobp** [araobp/acoustic-features#17]
I have been using scale API of sklearn.preprocessing module for normalizing data set for training CNN model. Is that the best normalization method?

**Comment by araobp** [comment]
scale() of sklearn.preprocessing seems OK.
